moduleframework/helpers/openshift_helper.py

```diff
@@ -173,10 +176,11 @@ def _verify_pod(self):
                  True, application is initiated and ready for testing
         """
         pod_initiated = False
-        for x in range(0, 20):
+        for x in range(0, 50):
```

Comment:
it is possible to have there some predefined constant, defined for example in common.py like:
``OPENSHIFT_INIT_WAIT = 50`` and then use it there.
Just fomal note. see example like: 
https://github.com/fedora-modularity/meta-test-family/blob/devel/moduleframework/common.py#L89